### PR TITLE
Update notebooks from the pulse-level control PR 

### PR DESCRIPTION
**Pull request by BoxiLi**
I rewrote the first 5 cells and changed the introduction of noise a bit (between cell 9 and 10)  of the introduction notebook. Other notebooks are just updated from the code.

### CODE REVIEW

**Comment by BoxiLi** [comment]
@nathanshammah

File: examples/qip-noisy-device-simulator.ipynb [hunk 1]
```diff
@@ -10,11 +10,14 @@
     "\n",
```
Comment (nathanshammah) [inline examples/qip-noisy-device-simulator.ipynb]: <p>Typos:</p><p> </p><ul><li>porjectQ ---&gt; ProjectQ</li><li>add space before (QIP)</li></ul><p> </p><p>You could break the first long paragraph in two. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qutip/qutip-notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (nathanshammah) [inline examples/qip-noisy-device-simulator.ipynb]: <p>for some reason this Latex equation is not rendering? maybe need to put on many lines</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qutip/qutip-notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

File: examples/spin-chain-model.ipynb [hunk 2]
```diff
@@ -34,27 +34,43 @@
    "metadata": {},
```
Comment (nathanshammah) [inline examples/spin-chain-model.ipynb]: <p>what about writing a script to detect qutip version...? I'm saying for the future. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qutip/qutip-notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (nathanshammah) [inline examples/spin-chain-model.ipynb]: <p>is this Bloch sphere rendering?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qutip/qutip-notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Comment (nathanshammah) [reply examples/spin-chain-model.ipynb]: <p>ok it is.</p>

File: examples/qip-noisy-device-simulator.ipynb (hunk 1 above)
Comment (nathanshammah) [reply examples/qip-noisy-device-simulator.ipynb]: I mean, just an if, that runs one block or the other depending on version

Comment (BoxiLi) [reply examples/qip-noisy-device-simulator.ipynb]: I cannot really see which cell you are pointing to. If you are talking about `pulse.print_info()`, I guess LaTeX only renders a `Qobj` if it is the last object in the cell. I'm using python print so I don't think they won't be rendered.

Comment (BoxiLi) [reply examples/qip-noisy-device-simulator.ipynb]: I find there is already a space before (QIP)?

**Pull request merged**